Lombiq.Privacy.Tests.UI/Constants/ElementSelectors.cs

```diff
@@ -0,0 +1,7 @@
+namespace Lombiq.Privacy.Tests.UI.Constants;
+
+public static class ElementSelectors
+{
+    public const string PrivacyConsentAcceptButtonId = "privacy-consent-accept-button";
+    public const string PrivacyConsentCheckboxLabel = "div.form-check > label";
```

Comment:
It should be `PrivacyConsentCheckboxLabelCss` because this variable contains the CSS query string for the label, not the label.